tests/test_conda_subprocess.py

```
@@ -42,6 +43,20 @@ def test_check_output_name(self):
 
+    def test_check_output_env_name(self):
+        expected_output = "py313"
+        self.assertEqual(
+            check_output("python --version", prefix_name=self.env_name, universal_newlines=True).split("/")[-3],
+            expected_output,
+        )
+
+    def test_check_output_env_var_name(self):
+        expected_output = "py313"
+        self.assertEqual(
+            check_output("echo $CONDA_PREFIX", prefix_name=self.env_name, universal_newlines=True).split("/")[-3],
```

Ruff (0.8.2)

Line too long (114 > 88)

(E501)